src/Components/index.js

```diff
@@ -1,23 +1,35 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
+
+import getContacts from "../Services/getContacts";
+import Contacts from "./Contacts";
 
 import Header from "./Header";
+import NewContact from "./NewContact";
 import Search from "./Search";
 
 const Component = () => {
     const [openNewContact, setStatusNewContact] = useState(false);
+    const [contacts, buildContacts] = useState();
+
+    useEffect(() => {
+       buildContacts();
+    },[]);
```

Comment:
I don't know why did you write this, but please remove this useEffect, because it's unnecessary